ibm-event-streams/README.md

```diff
@@ -89,9 +90,9 @@ extracted Apache Kafka installation.
 
 # Configure `kafka-connect-cloudant` as a source connector
 
-Configuration for publishing messages from Cloudant to Message Hub using Apache Kafka and `kafka-connect-cloudant` as a source connector.
+Configuration for publishing messages from Cloudant to Event Streams using Apache Kafka and `kafka-connect-cloudant` as a source connector.
 
-1. Download the [`kafka-connect-cloudant` jar from Maven central](http://repo1.maven.org/maven2/com/cloudant/kafka-connect-cloudant/0.100.1-kafka-1.0.0/kafka-connect-cloudant-0.100.1-kafka-1.0.0.jar) -this tutorial used `kafka-connect-cloudant-0.100.1-kafka-1.0.0.jar`.
+1. Download the [`kafka-connect-cloudant` version 1.0.0 jar from Maven central](http://repo1.maven.org/maven2/com/cloudant/kafka-connect-cloudant/0.100.1-kafka-1.0.0/kafka-connect-cloudant-0.100.1-kafka-1.0.0.jar).
```

Comment:
Because of the two versions involved here this statement is confusing. The kafka-connect-cloudant version is still `0.100.1` for use with kafka `1.0.0` APIs.

Reply:
Fixed in ee9fdf1